TLM/TLM/Traffic/Impl/SegmentEnd.cs

```diff
@@ -1,287 +1,358 @@
-#define DEBUGFRONTVEHx
-#define DEBUGREGx
+// #define DEBUGFRONTVEH_off
+// #define DEBUGREG_off
+// #define DEBUGMETRIC2_off
 #define DEBUGMETRIC
```

Comment:
not used?

Reply:
Removed all unused defines in a later commit, so just a couple left